src/test/scala-2.11/markets/participants/TestOrderIssuer.scala

```diff
@@ -28,32 +28,32 @@ import scala.concurrent.duration.FiniteDuration
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2363386119/issues/source?bid=2921361&fileBranchId=3219434#l28)